src/functions.js

```diff
@@ -0,0 +1,126 @@
+import storage from '.';
```

Comment:
- [ ] It looks like there might be a few errors while running your project. No worries though, let's work together to get everything running smoothly as you expected. The problem could be related to the way you're importing the storage from the index file. To prevent these errors, try exporting storage before importing it correctly. Once you make these adjustments, your project should work just fine.

   ![image](https://github.com/ulises2607/ToDo-List/assets/107952724/2aa63013-07d1-4fcd-9d2c-d4f67098e4c4)

- [ ] If I close the previous error, I got another error when trying to delete, it could be related to the import issue mentioned before.
   ![image](https://github.com/ulises2607/ToDo-List/assets/107952724/546d06cb-ae2c-4921-a6cc-d3e21b314414)

Reply:
Ok thank you! can you give some suggestions? @taldr27

Reply:
You could consider separating your functions in different files then you could export your storage as a default, and import it correctly.